KEYS.md

```diff
@@ -0,0 +1,24 @@
+# Keys
+
+The Linnia protocol need 4 different keys to work: (Ethereum Private Key, Ethereum Address, Encryption Private Key, Encryption Public key)
+
+
+
+### Ethereum Keys
+
+In order to add data to the Linnia Smart Contracts, the user needs an ethereum wallet to be able to sign transaction and add them to the ethereum blockchain. (We recommend to download Metamask https://metamask.io/ to create the **Ethereum Keys** and sign the transaction)
+
+There are 2 different Ethereum Keys, the first one is your **Ethereum Private Key**: this has to be private, nobody should have access to this but you. With this key you can sign transactions and add them to the blockchain. This is you authentication key, to prove your identity.
+
+The second one is your **Ethereum Address**: this is public and is how external users refer to yourself. If someone want's to share data with you using Linnia they will need your Ethereum Address.
+
+
+
+### Encryption Keys
+
+In order to add data to the Linnia Protocol, the data has to be encrypted. That means that the data will look like a bunch of random letters and numbers. When you encrypt some data you are hiding the real content so nobody can understand. In order to recover the original data you need to decrypt. There are also 2 **Encryption Keys**
```

Comment:
Remove the sentence about random letters and numbers maybe?? I think the next sentence covers it